dispatch/dispatch_test.go

```diff
@@ -19,400 +19,256 @@ var (
 
 func TestGlobalDispatcher(t *testing.T) {
 	err := Start(0, 0)
-	if err != nil {
-		t.Fatalf("received: '%v' but expected: '%v'", err, nil)
-	}
-
-	running := IsRunning()
-	if !running {
-		t.Fatalf("received: '%v' but expected: '%v'", IsRunning(), true)
-	}
+	assert.NoError(t, err, "Start should not error")
+	assert.True(t, IsRunning(), "IsRunning should return true")
```

Comment:
I would like to request a `require` pass for this testfile and check if any of the `t.Fatal` changes should be updated to `require.NoError`

Reply:
Not massively convinced here.
I think it's marginal to continue on in `TestGlobalDispatcher` to test `Stop` if `Start` failed.

That said, I'll change it anyway.

Fixed a94d4b53
Fixed cb3e5ab6
Fixed 297d10ac - Simplify TestMux

Shout if you're happy and I'll autosquish them down.

Reply:
> Shout if you're happy and I'll autosquish them down.

https://www.youtube.com/watch?v=Sdr3cpqQsSY 😄